ebp/doc.go

```diff
@@ -34,15 +34,77 @@ const (
 	CableLabsFormatIdentifier = 0x45425030
 )
 
+// cableLabsEbp is an encoder boundary point
```

Comment:
Can these type and interface definitions be moved to ebp.go?  We want to move away from using doc.go like this as it should only be used for documentation.